apps/src/lib/climate-variable-base.tsx

```diff
@@ -482,6 +484,18 @@ class ClimateVariableBase implements ClimateVariableInterface {
 	updateMapWMSParams(params: WMSParams, isComparisonMap: boolean): WMSParams {
 		return params;
 	}
+
+	getForecastType(): ForecastType {
```

Comment:
Question.

Any thoughts about using [getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) instead of a method that isn't doing anything (i.e. no action, verb, etc.) but instead returns value of something else.

```diff
-  getForecastType(): ForecastType  {

+  get forecastType(): ForecastType  {
```

Maybe we just keep as is, since it doesn't create a dissonance between the new code and current.